src/windows/BarcodeScannerProxy.js

```
@@ -243,7 +246,19 @@ module.exports = {
             // rotate the preview video
             var videoEncodingProperties = capture.videoDeviceController.getMediaStreamProperties(Windows.Media.Capture.MediaStreamType.videoPreview);
             videoEncodingProperties.properties.insert(ROTATION_KEY, rotDegree);
-            return capture.videoDeviceController.setMediaStreamPropertiesAsync(Windows.Media.Capture.MediaStreamType.videoPreview, videoEncodingProperties);
+
+            try {
+                return capture.videoDeviceController.setMediaStreamPropertiesAsync(Windows.Media.Capture.MediaStreamType.videoPreview, videoEncodingProperties);
+            } catch (e) {
+                if (e.number !== PHOTO_SEQUENCE_IS_IN_PROGRESS) {
```

Could you add a comment clarifying why we can safely ignore this error?